test/tests/dirackernels/stateful_point_source/tests

```diff
@@ -1,7 +1,8 @@
 [Tests]
   [./stateful_point_source_test]
-    type = 'RunException'
+    type = 'Exodiff'
     input = 'stateful_point_source.i'
-    expect_err = "Error: Stateful material properties not allowed"
+    exodiff = "out.e"
+    recover = false
```

Comment:
`recover = false` is not an option for something that is core functionality like this.  If the design can't be made to recover... then it needs a different design.